proposals/lip-0046.md

```diff
@@ -281,22 +286,23 @@ After all the transactions of a block `b` are executed, the following logic is a
 
 where the `isSeedValidInput` is the internal function specified above.
 
-### Block Creation
+### Block Generation
 
-#### Block Initialization
+#### Assets Insertion
 
 The asset data created by the Random module contains a serialized object following the `blockHeaderAssetRandomModule` schema where the property `seedReveal`, the seed revealed by the block generator, is set by the module. This can be done by implementing the hash onion algorithm introduced in the [Appendix B of LIP 0022][lip-0022#hash-onion-computation].
 
  ```java
 blockHeaderAssetRandomModule = {
     "type": "object",
+    "required": ["seedReveal"],
     "properties": {
         "seedReveal": {
             "dataType": "bytes",
+            "length": 16,
```

Comment:
Maybe add this to the constants as well? No big deal tho

Reply:
Done